app/assets/stylesheets/landing_page.scss

```diff
@@ -0,0 +1,86 @@
+.overlay-container {
+  position: relative;
+  overflow: hidden;
+  height: 23rem;
+  padding-left: 15%;
+  background: #000000;
+}
+
+.overlay-container .overlay {
+  position: absolute;
```

Comment:
Properties should be ordered bottom, color, left, margin, padding, position, right, top, width